Admin combi + winkelwagen

1058989 opened the pull request:
admin spullen gecombineerd met wijnkelwagen.

1058989 committed:
admin halverwege progress

Admin in staat om de options (materiaal) te bekijken in een tabel. ook optie om het in detail te zien. Ook kan je het verwijderen (en bijna in staat om aan te maken). wat dummy data voor de user tabel en de offerte tabel want die heb ik nog niet. ()

1058989 committed:
Merge remote-tracking branch 'origin/master' into admin-Combi

Lvdek3 committed:
Wijnkelwagen shit

luuk180 committed:
de berekenbestelling pagina werkt

luuk180 committed:
de vergeten vensterbank toegevoegd aan het totaal

luuk180 committed:
loguit knop

marieclairex committed:
update login

1058989 committed:
admin big update

admin is  instaat op een materiaal/user  te zien op de tabel.  De geprinte strings zijn nu in het nederlands.
Op de tabel page is de admin in staat om
1. zie detailed info van material/user
2. verwijder material/user
3. edit de gegevens van material/user
4. maak nieuw materiaal/user aan. tot nu toe kan iedereen de admin page in.  ook heb ik admin-add-materiaal en admin-see-details-materials verwijdert want die functionaliteiten zitten in admin-see-materials

luuk180 committed:
marie-claire d'r verkrachtingen ongedaan gemaakt

Lvdek3 committed:
Zelf invoeren afgemaakt

luuk180 committed:
css gefixt

luuk180 committed:
Ik weet het niet meer

luuk180 committed:
Kleine wijzigingen

1058989 committed:
Fix create quatiaition

nu zijn data in database niet meer 0. maar gewoon inputs

1058989 committed:
Merge branch 'Wijnkelwagen' of https://github.com/teammaxxter2-0/Web-applicatie into Wijnkelwagen

1058989 committed:
Merge branch 'Wijnkelwagen' into admin-Combi

Lvdek3 committed:
Luuk wizard

1058989 committed:
update to admin

wat dingen verandert aan admin voor de pull van wijnkelwagen. nu klopt het. al even een test gedaan met pdf'aanmaak

1058989 committed:
Merge branch 'Wijnkelwagen' into admin-Combi

luuk180 reviewed (changes requested):
Zorg voor live offertes ipv mock data

1058989 committed:
token

iets met token gedaan.

1058989 committed:
admin en wijnkelwagen

username: admin, password: admin om als admin in te loggen. offertes worden nu geprint in admin tabel. en toast bij inlog. kan ook pdf genereen maar ziet er basic uit. + wat locaties van files veranderdtdtdtdt

1058989 committed:
fixed error page

stuur nu naar goed naar de error page

luuk180 commented:
Iesh grote blij
